wrangler-core/src/e2e-test/java/io/cdap/plugin/wrangler/actions/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Package contains the actions for Mysql features.
+ */
+package io.cdap.plugin.wrangler.actions;
```

Comment:
End the file with a blank line, to avoid check style errors. Check all the files.